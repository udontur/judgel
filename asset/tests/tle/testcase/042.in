3927 55
71550 50095 48635 80816 42894 35123 88368 4733 7547 65039 25790 42632 28981 95147 87455 67875 13756 39024 78212 65105 568 4685 54720 16951 95899 42639 32261 31060 26844 62159 1148 8268 10824 82105 4775 20379 81766 66363 26743 98545 76870 81078 51064 24283 50745 49388 35360 51814 77179 99648 96278 61561 24943 68398 26402 8408 76036 14043 79773 24039 57588 41574 97876 18845 72688 1829 56505 43225 93362 53608 57895 49886 83203 35754 26756 66899 43483 18593 21531 17945 85024 97179 76957 96471 56906 48543 63519 25146 67262 26533 95228 99574 76622 30110 31255 98814 94688 67901 39713 63966 35432 80008 72996 18625 63744 8042 16483 50366 21975 47854 42963 51638 38666 69560 30618 11153 75134 35136 71382 93870 90530 82359 46611 46221 65339 4527 48351 58193 52761 79037 59532 62458 44798 81081 90214 9417 38441 9112 78778 87250 47432 77785 10131 79413 4550 34436 71819 22819 85131 39086 44882 96769 70148 31592 6078 50868 82651 48822 22364 9694 6945 49834 78714 99544 55726 88828 36447 19333 41386 63882 64398 34736 55567 74110 55619 92963 66187 23949 77048 24516 43492 49259 8015 50467 23775 27293 94117 97407 55421 32154 95451 21191 26818 62460 25202 34104 67200 58721 52578 42268 61918 54838 50974 61134 4994 99318 60051 95252 74234 22620 99817 64848 46854 45872 73227 43630 26445 6783 98359 32249 12793 74848 50723 97049 62171 8739 50730 4698 32447 54180 92610 81420 6572 75543 98014 56412 80219 7322 67535 65401 77054 70216 50938 55178 89227 43392 49543 6394 15186 2411 64602 39027 93899 60374 41195 86233 78382 55338 75510 66492 58927 73701 86754 93509 5895 6051 28633 14038 70303 12790 67447 25981 2338 30850 22098 75117 65654 11084 98505 6359 14686 37251 91486 41418 94319 4858 5435 60953 17552 56139 20602 46289 97945 60063 70469 22188 29435 5472 2052 21019 70839 34588 56313 49887 70061 69560 21724 761 86833 43184 57235 79321 24762 62534 32129 88917 38060 62106 80162 48670 69831 62659 29607 21073 2948 26889 7116 65434 79197 2330 77316 7845 8258 77348 67429 58303 37425 29567 74016 64723 53590 45940 4337 97810 85363 86808 54861 60462 97681 3326 26630 69973 29310 51251 32556 5278 23200 10736 75862 84019 65010 95217 59865 93702 29706 13570 51135 64235 97238 87445 59383 93178 41813 51957 28690 10853 44099 78042 20038 60397 47419 43408 54941 42325 13273 81164 83779 32906 3114 11228 59252 4844 87307 5384 9529 97383 81389 12444 65548 8345 47772 79689 70293 38729 83366 45707 18333 73177 33079 790 66129 44468 74823 83759 10630 17009 1625 35163 41027 13199 71766 47466 27718 71753 56669 74613 54472 61363 86542 54716 18686 31274 46315 96366 49775 78611 4548 94673 22916 91954 64222 59330 69115 76375 85760 25556 37939 46290 73523 37737 93862 36273 97995 39780 63751 57742 23782 16511 4190 21879 5839 25051 60763 23892 47906 9960 55737 69117 88920 26017 16982 57831 66291 75836 67616 16063 57752 52633 77210 18261 41260 13143 71514 35112 36073 89299 74221 7181 28587 36667 75194 98285 35841 18154 73716 62621 33368 31784 4143 99252 43302 42558 81104 44567 78441 27173 38222 87498 21856 47258 17956 14687 22739 75508 74923 69961 46431 88509 55707 50587 55324 74178 82387 64035 70384 91042 64078 15228 4164 25696 18702 198 72549 1574 34078 74185 67223 82518 87737 17912 15708 93038 50843 69846 71397 33464 21706 40668 80449 56276 33802 93968 62183 53721 27403 58982 27418 55143 36797 19807 33522 71931 97978 64277 61376 9152 8776 71381 14379 87465 77807 76789 48821 41895 66671 53196 57473 28667 47721 20345 75009 7153 47872 80365 73623 82033 53918 90394 98443 55996 51990 62205 91519 11382 11330 46083 57214 86064 24784 4491 33045 46955 52037 42599 67360 46361 44231 1167 61953 17960 50024 60553 62740 71567 74647 33569 99715 52251 40255 94769 41722 24990 41922 97946 15380 57385 11179 20982 57598 79907 69326 55214 91298 37916 37442 3169 49747 66531 53097 68904 7183 56841 64753 60362 69232 14073 82222 94708 69497 39400 76404 84989 91001 65665 23884 11359 48112 39398 69934 74952 14028 2593 96384 88347 73832 78077 44156 52238 8276 26416 67217 83308 41714 49033 15996 71705 7791 57736 19455 55738 99381 63114 67784 92246 20555 36032 61542 7648 27575 61072 34270 59170 6127 87691 24821 16686 37039 63512 2594 45463 24907 72606 27108 98783 588 90593 8483 170 96852 26562 31760 68062 97636 56230 91843 51717 79762 75122 74982 54066 45944 34167 18612 15348 90644 47623 37763 12756 31652 72954 63938 35659 41361 47504 39935 14277 8001 51640 57163 81448 47150 67600 3936 22904 66580 42061 1246 9686 69582 63917 42914 33550 27678 46381 34023 13596 45435 44743 67988 13983 67025 91468 67328 87960 17428 45995 94256 48138 54138 74132 86475 60325 88049 6569 14888 39056 76244 90903 44302 50346 20231 28800 7450 55979 35224 24585 24416 9118 37141 6570 18196 88093 67511 18052 94732 50703 87574 65012 40708 10808 39723 46291 54672 32929 87620 88117 13359 87886 68954 99125 46854 49076 75100 23968 13276 30464 7055 95952 76107 63900 91264 30158 50701 53010 80122 74901 46515 50556 95644 73154 45041 40057 69858 24864 73809 67579 93418 23193 77438 91846 29924 87909 29468 35007 55481 83136 86595 59368 59993 64795 14522 71405 57872 57462 30579 71857 94476 75628 66551 17357 22494 89625 71539 81032 6112 15653 93679 44551 44829 27618 53062 14053 28418 71145 12379 38665 87152 11502 17361 86667 41943 77769 94614 33359 87495 47380 90062 70498 91153 54380 25473 19220 81165 22301 58803 35769 11830 33723 5278 93096 87908 56958 89667 7478 36540 75802 71315 68032 56528 7569 51951 43564 89187 88393 70588 40726 42808 93101 98138 85677 84147 14882 36860 26262 82191 28602 94432 76016 46831 10823 37522 96063 65767 95831 7651 88303 87283 26536 74906 7440 13340 77253 73995 13580 52665 58734 90149 54383 46948 26804 80997 84374 83095 94704 34709 57718 66065 3350 35353 97654 77356 23956 23859 84468 70224 19555 71304 7396 64509 97516 39049 97526 94364 89820 74350 33815 99188 94194 5364 15626 61215 74035 48732 86655 74307 62499 72943 60819 3774 64309 48986 14262 1273 17252 21419 48916 65146 56779 20925 31089 46581 48552 82277 91128 36324 46615 3728 74015 78704 42798 26429 10371 93794 40279 94290 2928 6975 67725 51973 47963 32396 84188 69254 14676 72832 13807 83998 52393 93042 77907 72672 43465 66379 30980 58987 55493 56338 27690 70137 47650 85325 85553 80306 22200 6271 6214 87655 33272 84393 27833 49749 87481 92392 5158 44040 44582 57497 42219 14768 6412 20194 90651 47422 34217 51770 19544 14485 80288 1239 4861 81457 67754 42982 80492 37966 89705 85505 1342 46157 87788 3744 7748 32590 89964 80284 66379 63350 75838 29756 5143 29354 3430 8209 58189 94548 78428 71787 16836 82768 14271 51787 11354 73527 85569 27185 53926 75605 30076 74607 20136 5359 72071 52176 36409 17647 34282 59806 88371 93590 4726 31273 57696 76416 24256 89500 95012 47500 15805 85363 33989 66809 94011 20804 37635 64672 36868 71789 91281 45291 50040 8921 24918 37339 29704 25354 92076 5806 6833 2332 9407 13929 21864 31676 30359 45264 61509 48125 84578 14680 18452 68183 34051 41664 47530 14439 5708 66278 64891 83702 80759 93091 68233 23045 4686 34754 29924 40528 50701 96319 4795 91832 14363 27885 97960 70455 82412 74039 3564 66862 82919 59341 46705 95702 1027 78386 64283 51144 92045 55941 32531 91722 84073 24403 48365 58142 82672 60924 67145 23370 70312 31834 15614 34147 70336 14459 84047 81829 90160 10622 35021 9955 80431 48296 48103 97703 88767 38952 34136 70686 49674 98032 41602 95026 96412 40761 2534 38921 62395 80537 98276 47999 77105 10093 38762 23783 68691 59260 93811 29315 3396 89507 1430 34545 40184 27864 80722 58940 97822 36911 79146 54556 47656 1999 98566 17699 53153 54528 61214 90114 34677 68168 20587 89924 17601 54458 14026 10240 87217 38291 85827 17447 84162 55236 37347 95178 59776 53064 5630 48094 72694 1298 41665 53484 10708 50199 80532 81943 31227 45962 20524 19662 14482 73321 77357 93353 14003 42474 91437 44881 62001 5011 61964 9849 12482 74566 26658 44114 69274 96418 51044 47260 11541 1064 26817 77552 38984 67753 45108 4579 403 19896 6865 17389 71500 63264 83935 16258 67304 67319 80423 39180 6518 82609 4642 13683 1974 97339 87801 86537 80768 22194 98737 50602 91944 85226 58818 23392 46062 28098 62066 68325 23295 43568 80611 55026 47994 89233 60777 99492 71008 47171 67765 93507 13445 2315 97826 44650 94806 50807 91532 1321 65730 56592 85337 26684 49069 24176 90783 81221 46701 25346 510 42995 82907 9960 74240 94389 39884 94095 38664 91646 42758 99861 39564 87631 85327 42629 33651 77947 51987 56404 51103 82 89104 49467 74877 13257 88642 25256 74516 88674 75279 48499 97098 33630 50211 15902 75158 40720 66174 4376 39564 24137 62915 67022 28911 7533 90773 3500 90293 34891 81714 28408 20286 9924 95159 93810 37775 57846 70087 33062 23614 35355 82701 87245 90670 87819 72641 54367 31415 62443 1409 37571 41347 14105 4516 9847 69740 21507 79128 67877 50258 43681 64499 61908 87486 60034 99630 10434 7308 68021 83071 41914 38959 98428 66543 93318 20333 79096 86886 70028 81880 71852 20674 88439 20107 2276 81012 98545 81912 35511 83862 34171 95382 71923 8349 88212 18125 89983 54353 74477 20417 53948 2669 29976 21979 47840 49806 17904 56049 69108 1459 70294 5401 18527 48581 67074 34833 63097 48742 11160 68163 99754 50086 62461 99864 70730 23797 45171 71934 7373 41261 60030 63767 69125 1128 55872 49278 25695 68800 95684 89209 21675 32474 4017 89883 87435 61729 95856 62145 8698 65317 48398 88344 91489 5757 65580 67302 19377 71856 75063 76253 75412 77088 11506 58691 95105 26668 8484 16122 39350 69389 21613 29286 80222 58525 26868 79486 73669 37088 10191 55586 73693 8767 73076 28342 91093 66930 37383 6662 42525 25818 30350 18091 89220 8524 14815 62364 31932 26656 80019 24646 15459 72124 82967 58863 49229 52709 40109 56083 28332 65743 64931 87389 62573 29611 64042 81058 15758 93175 47888 89408 70398 68412 46833 54149 52596 68933 11386 65414 93660 30572 34732 38490 20346 99581 95960 65146 59893 77708 10238 25707 57940 35114 79801 11633 32079 82400 4970 49983 58400 14461 93643 25319 53251 12559 36694 50674 35232 58378 83916 49816 30270 69276 47051 19029 28262 3245 64590 29541 63479 82717 39540 66351 60911 92174 7515 15262 87539 53768 53950 78300 49586 63409 97078 30726 92189 14627 72644 76901 93629 15806 81496 32714 2889 25027 96125 63902 76101 58910 96375 20396 93199 10373 14619 55515 74947 72605 30693 13238 19623 52967 73487 36289 14625 58646 2345 61266 60888 32000 82377 50747 31326 42806 17551 17147 83197 89741 79522 65003 35784 7092 33010 25734 21186 98715 6095 310 79671 98038 72398 13687 48212 52599 8471 76238 59062 96855 95934 75714 32903 9535 51899 26409 35105 49774 88005 64986 38574 2317 43611 77217 23614 20339 19491 86095 41352 12142 19138 42812 60154 33743 9630 55935 23692 68905 81894 3769 91919 87496 53967 18183 76266 33498 54099 88989 53178 56609 66651 34092 99521 35685 51457 48058 72068 36803 66442 89920 35330 72364 12117 84091 22940 58214 98909 88263 42634 52655 34629 12581 86869 74950 96487 36035 44693 90935 38963 12562 12030 23199 33544 41994 97425 55528 8675 17633 67355 6551 63797 76674 21239 82048 62018 81953 76252 33715 57588 49935 50461 49496 5274 5160 40524 3244 14562 98705 89151 69977 88438 72636 18539 13382 4154 59527 39988 28897 23075 56837 25401 15294 45217 46068 97788 13927 34841 22427 97249 51472 15582 31444 23956 52755 77438 47106 73568 57741 19266 72782 39014 17677 8001 36376 44937 84632 72519 59122 38918 43192 52453 92088 336 45402 61653 76562 39826 32753 96645 19425 31817 21370 67855 47541 41929 78846 25798 85957 85402 23521 86702 50000 60468 55132 37242 76327 32829 27319 50319 87045 52274 74815 42953 98977 10823 80929 50036 18294 2681 31337 20175 72176 70377 42542 75491 39462 68906 73326 31312 56539 4133 74601 5970 66304 56376 40687 93148 75064 35646 29693 98499 23390 83748 72135 30444 87530 84349 4438 61188 93184 88122 9331 41862 79716 26555 29753 53048 35042 41302 57229 68918 93475 44707 79289 39466 28334 40533 92264 91815 79830 89434 99447 35637 1231 32725 64012 88725 70419 32878 52383 42601 24281 64533 21698 33105 84364 57287 64458 95420 90921 43995 43213 38116 53738 33591 26270 36115 37478 52582 66939 20348 43641 83968 57088 17932 56711 77380 12783 69955 15475 92419 14723 4840 24188 13840 96879 33964 42265 14227 32550 28638 12402 40476 16392 39635 40038 26256 38471 82336 63549 35476 82221 90221 45490 86282 37688 28767 85737 50138 76492 88340 32340 76461 90829 93416 19747 73966 73052 37765 53525 4485 44302 79201 35452 24666 39505 64705 27845 48134 12208 39435 30566 82139 2994 12328 66146 61807 5518 14451 78173 16586 55259 46399 94725 81530 71873 51212 5607 43102 72339 87013 55487 10189 43 78186 44538 62587 45976 80064 67908 2542 69034 59733 56937 70895 28635 37358 81463 26920 81996 27215 7899 79259 70856 36359 17043 53320 99430 33731 9490 37026 76800 65356 49946 31857 99141 74320 19645 31921 20410 92099 66067 39548 30828 69758 44819 42752 17629 94606 43126 53656 21765 71941 19321 84124 22927 2453 44994 19694 54776 80820 90335 8207 88244 5355 36304 50450 71323 99414 68001 72156 21729 71753 34155 32604 90599 94495 78105 99157 77702 10193 94521 36080 96529 31886 5501 26296 22740 33276 12698 52652 21789 8725 29939 139 40386 84434 34415 28446 77314 73943 75622 80257 37400 14836 40098 33652 37596 21881 48698 99856 83908 24051 65243 54681 2723 94164 92330 9041 90286 96225 92770 26454 7794 86615 19859 61383 66606 4769 99246 2439 70604 66671 690 15596 39221 54981 20397 66302 43316 28920 90822 28539 63697 4188 73512 50461 17297 51080 47812 85888 1021 78769 87472 28892 66527 47810 34627 75965 84219 80767 13460 35424 25471 48135 59491 75680 35798 45956 3222 61825 64118 97249 76709 43025 67208 40898 10527 63947 92295 80384 95646 90181 86026 90878 87318 28602 60395 19905 92499 43567 56231 9307 64067 37155 55363 45882 80804 40581 19298 4096 30313 59209 36486 30017 77686 73232 38781 70948 82032 56784 15634 38813 35267 38034 72742 44698 97702 30134 21338 75505 48274 37271 9072 25929 98090 45682 46860 12700 49162 45267 9243 56496 9680 42196 16333 54975 80514 21336 65956 96449 64209 61113 90534 49400 64123 83628 23196 14819 1401 26080 17539 9908 52934 34085 4033 693 59471 93594 45135 78119 45884 66466 2672 26952 33702 15227 87349 86076 1481 2305 50561 46146 70357 66412 43229 68518 47381 85842 36003 23460 83050 26913 38007 61709 83845 58137 73296 76029 46155 46857 70861 11738 81137 40342 2828 11018 19081 72074 14732 4812 27499 386 87131 99821 50449 44891 87643 5358 51698 4837 96808 81802 33088 56947 19729 48575 92335 9204 89024 89490 34521 73985 73587 2800 979 8403 40199 13928 21064 79962 45790 70492 688 38705 82715 26423 41053 18295 18062 12674 77299 12415 6834 64636 24425 39654 60525 21767 97777 59209 55618 58080 25761 90429 27949 73046 84930 95205 4440 32681 4557 25185 89544 21788 48837 72972 69979 11729 15220 79700 13415 76534 2478 60735 15395 47654 51756 25258 7844 16546 24399 87865 21141 62357 96315 46333 103 28060 9171 32648 53126 89432 99553 56978 13751 25232 64515 55357 80666 79905 97100 62492 16585 4199 19311 27327 62453 28057 29761 81982 88747 7103 88715 7037 72549 25556 94320 49929 7126 50189 66394 76236 41393 98870 91013 94797 92193 40112 40945 34453 50106 6175 12214 39348 98096 31063 7321 50422 63309 79817 50281 32941 72541 21602 19443 4603 700 1322 14958 96056 19401 86345 90588 69262 83791 16862 6015 31978 15336 70740 76067 63330 70773 55392 66195 59237 32431 5806 39839 5137 21053 86708 77538 13376 17082 27588 35085 45358 17651 75962 50843 159 88231 75697 88433 91068 43754 3716 30776 91074 47802 58124 94718 91549 8452 48048 76837 3409 12696 14086 76132 80776 1904 44971 92224 37123 32848 17765 68977 18656 70968 92657 82021 27741 82584 46887 29400 62540 77807 22974 72745 45862 72523 70294 7560 15094 71575 39343 31913 59148 84390 6501 68924 40620 97488 23230 64652 48924 1966 73388 91092 17266 571 75585 67212 7200 79588 9461 27341 38640 45536 10272 41244 68199 23888 29080 74495 72138 11115 49410 76381 57333 74359 39425 31824 28598 86479 44189 15623 7603 62737 10780 74739 46254 97393 77059 89021 29132 93404 83649 88675 17875 5453 84687 3256 94767 92635 30172 37049 61468 52603 74219 86358 10920 30409 30247 66430 48032 57647 14946 43675 38541 41044 72614 45528 57603 58138 91643 94089 46450 67378 73616 71710 95892 29570 59649 66614 12466 37378 48423 91148 98568 76076 65226 65823 98824 74177 96843 66261 69190 84116 35869 16585 33368 27154 19128 92900 24215 96465 56822 8739 89117 57864 88553 45181 74008 36131 83807 51618 54751 70792 16722 38971 29648 49468 36112 67533 78663 52005 94799 13490 70921 36109 19408 21837 83616 11146 9901 7867 54498 50197 1525 93833 52135 76186 59754 62912 54328 20729 42661 67734 35335 10011 75775 54253 24564 39311 9876 85394 27894 88808 86561 96204 40945 36507 65001 79661 4797 82156 48372 26931 75643 64820 81657 63600 49330 65903 39614 7691 59362 94719 29757 77855 6251 76680 80220 64477 50422 54039 43794 58909 21145 73277 24564 87958 81812 84910 21001 17708 61268 55571 15555 88175 28881 37988 12062 57329 52157 50784 22843 68845 12788 99991 91407 2382 84068 80274 43127 87974 94046 90908 41892 17692 45506 25348 23975 72661 37409 55422 89102 23431 98466 40822 31601 53107 7707 5139 88089 1351 7108 59885 82838 26769 47502 84053 98414 51760 7550 28810 30582 66037 35454 82971 26225 1810 21131 99578 82393 38949 29248 77418 18566 11098 8839 56437 46569 79933 35600 67303 48766 55748 93517 89922 25242 48049 17715 69827 68985 76610 22033 95545 99438 53823 76138 62228 9731 28519 49106 74558 33725 50855 29488 68405 1734 22531 39051 12878 45534 77737 59876 74553 58706 5222 24021 47366 86288 41267 66766 99322 25405 87673 55410 38407 72203 20327 44015 47066 89845 51087 56204 4979 40293 37780 71344 42692 11021 19993 23401 45549 49793 84820 76236 24363 80895 52222 55983 91499 14925 5940 18278 17782 95355 22339 8039 52894 31711 76027 13791 27416 16197 55266 41864 76640 29273 46230 51700 47834 70152 23822 16705 56285 81358 63546 7913 41573 87223 8133 4511 68940 28398 489 40241 29424 62766 78075 66497 57288 59569 55025 1692 80835 39920 18792 11765 58938 746 95708 80754 6746 25716 54702 59474 53072 99502 21116 25387 46158 48213 36077 18218 77105 11815 355 13663 41885 4268 4900 12601 76741 43992 34940 38411 98372 75270 52509 91652 63219 91377 76466 41663 79834 55962 8029 21170 14542 29516 26337 96088 70951 11270 61494 33533 23243 68159 52985 80301 17176 992 6173 15809 79896 9450 48595 54339 33540 24684 44534 28072 13289 57020 1540 33484 46690 70680 24190 66944 6576 72051 2298 91886 13049 15486 50645 89810 22463 55214 35623 78974 91069 914 89669 71514 62059 4888 43095 69878 79747 38035 83110 80842 90197 100000 75883 66722 56951 56741 86526 60806 42663 55150 26423 77728 33707 54734 78816 21607 95122 3909 411 6956 37973 32853 54867 14685 11143 15946 38855 68416 11805 46001 76016 38459 82149 39735 67888 77551 1851 50650 35806 57515 39199 87230 29512 96371 11063 27853 57805 32765 66018 47294 53243 37862 17434 16535 20944 93345 28904 18036 81894 53835 21763 19234 69719 31055 84101 86072 59834 48138 31185 27207 84449 32573 41388 71416 99372 8579 62871 52968 29407 16136 4838 324 53089 84475 26016 63394 30928 61140 52376 69496 42017 40419 99305 66603 68518 76439 17437 54857 59094 98629 49949 94675 41495 62244 80440 38160 56030 77490 88448 25790 15515 39566 57101 35673 79782 2529 47655 75316 41441 55536 25150 19580 58414 82465 18217 52372 31656 92498 15413 11924 43186 23418 55102 14855 66312 93088 43786 49631 11888 99600 34223 55198 20424 15065 56359 66068 84212 15738 22415 51848 4177 67919 70402 49633 81811 90958 36880 26370 22899 11302 23044 79847 60564 12046 92301 83965 12462 75712 63509 20062 83984 62301 65562 23055 14092 41655 57316 51857 92860 1338 13713 25366 2001 75399 70363 25419 95526 72667 9793 2825 80823 61933 27089 41536 25740 31880 97339 87591 23296 59990 96390 17031 40449 1661 98330 26816 53297 96782 54663 10299 94101 90518 65031 350 49336 83611 94288 18008 97654 81416 39639 88281 42795 12152 64496 31705 78755 8969 53627 35553 52658 65900 11789 46032 36958 60841 76407 92170 2536 49806 21408 2368 27836 10691 40069 76698 66979 49966 61005 1883 53467 47023 65166 19174 65361 65784 60261 56606 86953 24387 26095 33595 51305 23551 51301 57922 72107 30207 6148 89142 60137 33712 57212 10801 44025 85414 88796 55094 8880 7730 58353 60341 99887 54799 62477 14846 97005 67678 38130 8167 95058 13889 25517 36893 57444 4959 4877 39218 69973 12480 58209 1018 73616 39286 11508 12129 78134 21727 7589 62386 61146 74449 44728 80744 40220 35163 22943 65409 82359 66162 55982 87615 72763 52061 1783 61147 42905 22780 15932 23103 63262 66571 92477 91467 45404 29543 25157 75365 77329 32530 74309 16277 57242 50778 11235 76903 63047 80521 34798 53651 98823 79564 26847 81017 82153 32190 60933 6133 1912 61402 72918 81856 66075 97809 50942 64158 20380 42994 8903 82382 51752 84433 71628 29957 10177 37612 49710 74338 90265 16178 63688 48468 62049 84679 51593 30234 95041 63524 79687 49229 72156 9192 44881 97577 35870 55431 88953 90172 54557 26195 88208 78382 68856 81570 14824 53867 84717 21726 13827 16082 68658 58982 84920 45740 35532 52747 97261 68725 44671 29855 85300 71615 28740 72063 42168 22053 72645 8299 56605 99191 42585 53633 61859 50285 65779 39356 76617 83019 2538 33271 24451 24072 68489 15502 72788 57414 47861 61517 61210 15787 35092 76225 87366 49156 66218 91695 56592 7315 91043 84940 755 70431 58834 79770 11879 69556 32533 51048 69919 54131 73005 43172 47694 74964 47696 5803 44426 26067 16936 24709 86115 35865 21168 24434 58780 24531 63280 91471 74092 39462 73404 6839 21857 54043 46655 29557 95418 93821 95230 19936 77100 58815 63672 51182 56882 23923 98058 72023 56951 81576 52538 38931 64691 15003 68609 42369 69083 95325 25963 64993 12015 18865 70387 85433 6516 28233 69371 15642 32547 87104 76660 31666 4831 15239 90152 17881 78561 75632 81972 59978 56296 80645 20511 41002 75087 33976 2378 34883 35115 31860 22949 90203 49329 98816 20781 50550 20612 89879 81036 5432 99765 75536 83590 8394 17563 85543 45780 40873 22739 50153 37847 62108 33426 62944 27668 50512 92772 36389 53112 51515 16809 43746 97626 69345 2198 76629 10600 6612 89302 95323 30345 35493 45940 95697 88858 28775 49986 98154 99389 31801 77653 96505 27944 67012 31437 40531 6697 14431 42897 83517 20520 13654 39846 65781 35493 42373 82982 87181 17463 89237 88549 500 23782 91813 20004 12193 58627 44301 23914 54010 65668 51048 38704 54523 51120 25493 44000 33169 87070 51214 1072 77808 94145 11932 99668 33925 75835 80738 96815 48293 58559 80471 33305 50976 4527 99201 58580 86846 53587 15821 74034 36484 41491 33045 87589 21381 26106 81503 32253 50922 11043 90357 47408 1587 31354 32597 32938 42080 52543 30575 11353 83486 59398 60897 69957 35361 9048 1794 17736 34950 19513 53322 22429 11638 97338 97529 98093 90697 60788 8626 58348 53768 81624 11398 4796 36574 70322 12579 53923 15993 81744 27318 19397 96919 48645 56952 14436 23785 91552 99361 1980 74231 35317 21679 53203 39099 78252 98070 26519 62712 53479 95825 90422 59332 14298 58444 49931 26592 56466 50282 19856 9529 73788 59888 69442 54012 15897 32076 39619 72203 22012 2049 7837 7215 62480 78271 29899 82490 2442 86520 51656 92806 85316 41181 85321 56713 16935 65090 81420 3602 44904 60329 26180 55191 67074 61190 23417 90619 18607 92447 30247 80272 88858 77533 85362 79444 84030 10273 18300 59244 11726 64771 48224 82552 34756 55459 69699 82697 34571 95383 6097 24824 15653 95939 39196 2719 99591 2524 13621 26610 44568 18009 38990 72705 59860 96110 8847 16065
76963 96430 69779 7864 14931 68908 1540 24911 17986 40027 49516 65090 74325 29992 62649 17514 48436 53011 32743 46639 46112 54928 15078 15381 23570 52261 56622 86005 45990 29326 50319 43445 65268 92798 65595 80366 57437 61607 19368 94778 61920 89265 83888 97097 96273 66822 3882 79580 66218 35120 43410 84036 38286 81803 31452
