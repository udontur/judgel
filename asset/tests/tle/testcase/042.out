13717273114358
